internals/cli/masker/writer_test.go

```diff
@@ -1,322 +0,0 @@
-package masker
```

Comment:
There was a lot of value in these tests. Let's make sure we don't lose that.

Reply:
Of course. That's an artifact of this being WIP.